docs/core/compatibility/framework-core.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
This article and the one above both have the same title, `Preview 8 -> Preview 9`. Yet they list different changes. 

Also, the description for each article says `Preview 7 -> Preview 8`